backends/gdx-backend-lwjgl3/src/com/badlogic/gdx/backends/lwjgl3/Lwjgl3Window.java

```diff
@@ -60,14 +60,15 @@ public void invoke (long windowHandle, final boolean focused) {
 			postRunnable(new Runnable() {
 				@Override
 				public void run () {
-					if (windowListener != null) {
+					if (config.pauseWhenLostFocus || windowListener != null) {
 						if (focused) {
 							if (config.pauseWhenLostFocus) {
 								synchronized (lifecycleListeners) {
 									for (LifecycleListener lifecycleListener : lifecycleListeners) {
 										lifecycleListener.resume();
 									}
 								}
+								listener.resume();
 							}
 							windowListener.focusGained();
```

Comment:
This will throw an NPE if `windowListener` is null.

Reply:
yeah, should be ok now